_data/harnesses/kafka-consumer-application/confluent.yml

```diff
@@ -0,0 +1,178 @@
+dev:
+  steps:
+    - title: Initialize the project
+      content:
+        - action: execute
+          file: tutorial-steps/dev/init.sh
+          render:
+            file: tutorials/kafka-consumer-application/kafka/markup/dev/init.adoc
+
+        - action: execute
+          file: tutorial-steps/dev/make-configuration-dir.sh
+          render:
+            file: tutorials/kafka-consumer-application/kafka/markup/dev/make-config-dir.adoc
+
+    - title: Provision resources in Confluent Cloud
```

Comment:
Would we want to rename this to:

```
Sign up for Confluent Cloud and Provision resources
```

Reply:
Ack, updated in https://github.com/confluentinc/kafka-tutorials/pull/786/commits/9617b5570ca83d20d037f966c33e66b1fa0ab0a5